src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1547,15 +1547,15 @@ void Convolution::initTryBrgconvFlag() {
     // Due to performance issue, brgconv will only be enabled by default:
     // 1, static shape(dynamic shape may change weights layout if the input shape changes and cause performance issue: 86948)
     // 2, support amx
-    // 3, int8 without binary postops when avx512
+    // 3, support avx512 except int8 with binary postops
     if (!isDynamicNode()) {
         if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx)) {
             shouldTryBrgconv = true;
         } else if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core)) {
+            shouldTryBrgconv = true;
```

Comment:
Just wondering does it makes sense to disable brgconv with binary post-ops for FP32 as well? Do we have corresponding perf experiments?

Reply:
Code updated: disabled brgconv FP32 if the model has binary post-ops.

The data on ICX:
Total model number | Overall avg boost(original) |>5% boost number(original)| >5% regression number(original) | Overall avg boost(binary removed) |>5% boost number(binary removed)| >5% regression number(binary removed)
-- | -- | -- | -- | -- | --| --| 
111|10.95%|46|14|12.20%|44|13